dashboard/config/locales/blocks.de-DE.yml

```diff
@@ -462,7 +462,7 @@ de:
             '"vintage"': Altmodisch
             '"warm"': Rottöne
       Dancelab_setBackgroundEffectWithPalette:
-        text: Hintergrundeffekt einstellen {EFFECT} {PALETTE}
+        text: Hintergrundeffekt einstellen {PALETTE} {EFFECT}
```

Comment:
nice to see translators taking advantage of the parameter reordering functionality!